frontend/src/App.css

```diff
@@ -1,8 +1,24 @@
+html, body, #root, .App {
+  height: 100%;
+  margin: 0;
+  width: 100%;
+  padding: 0;
+}
+
 #root {
-  max-width: 1280px;
+  height: 100%;
```

Comment:
is there any specific reason for adding this `height: 100%;` to the `#root` element id?

Reply:
I added it while trying to add css to the page, but now i have removed it.